### DISCUSSION TITLE
Integrate ice40 viewer

### DISCUSSION THREAD

**Discussion by Jesus89** [FPGAwars/icestudio#852] [Ideas]
https://github.com/knielsen/ice40_viewer

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
